### PR TITLE
Minor performance improvements

### PR DESCRIPTION
**Pull request by Orhideous**

### CODE REVIEW

**Comment by primetoxinz** [comment]
Nice! Thanks.
As for the blasting oil, we can probably make it tick less often to help lessen the performance impact

File: src/main/java/betterwithmods/module/tweaks/MossGeneration.java [hunk 1]
```diff
@@ -32,7 +32,7 @@
     public static int RATE;
     public static boolean DISABLE_VINE_RECIPES;
 
-    public static void addBlockConversion(Block block, IBlockState mossyState) { //TODO: Could be the new block meta ingredient possibly
+    private static void addBlockConversion(Block block, IBlockState mossyState) { //TODO: Could be the new block meta ingredient possibly
```
Comment (DaedalusGame) [inline src/main/java/betterwithmods/module/tweaks/MossGeneration.java]: public pls

Comment (primetoxinz) [reply src/main/java/betterwithmods/module/tweaks/MossGeneration.java]: Yeah, this is used for add ons

**Comment by primetoxinz** [comment]
Ok, looks good.

**Pull request merged**